devel/ncurses/Portfile

```diff
@@ -44,7 +44,11 @@ configure.args  --enable-widec \
                 --with-pkg-config-libdir="${prefix}/lib/pkgconfig" \
                 --disable-mixed-case
 
-if {[string match *clang* ${configure.cxx}]} {
+platform linux {
+    configure.args-delete --disable-mixed-case
```

Comment:
This configure argument's name and description are confusingly misleading:
```
    --enable-mixed-case
        Controls whether the filesystem on which the terminfo database resides
        supports mixed-case filenames (normal for UNIX, but not on other
        systems).  If you do not specify this option, the configure script
        checks the current filesystem.
```
All Mac filesystems support mixed-case filenames. For example, you can have a file named "Portfile", which is composed of an uppercase letter followed by lowercase letters—in other words, mixed case.

What this configure test actually means is: is the filesystem case sensitive? If not specified, it checks at configure time. Our build servers have case-sensitive filesystems, but most of the users installing our binary archives do not, so we specify `--disable-mixed-case` (which should be named `--disable-case-sensitive`) to tell it to assume the filesystem is not case-sensitive even though it is.

Does `--disable-mixed-case` cause some problem on Linux? Since as far as I know it does not cause a problem on case-sensitive Mac systems I would not have expected it to cause a problem on case-sensitive other UNIX systems. If it did cause a problem, depending on what problem it caused, maybe your change here is correct, or maybe it is correct to omit this flag on all non-Darwin systems.

Reply:
Thanks for the detailed explanation @ryandesign.

For some reason, I can no longer reproduce whatever error this fixed. I don't remember changing anything...

If this fix was at some point needed on both @RJVB's and my Linux machine, maybe it is still best to omit this flag on all non-Darwin systems? I'll push a commit updating this.